pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -727,20 +727,12 @@ func getDefaultKieServerImage(product string, cr *api.KieApp, serverSet *api.Kie
 	return api.ImageObjRef{
 		Kind: "ImageStreamTag",
 		ObjectReference: api.ObjectReference{
-			Name:      getName(imageContext, image, imageTag),
+			Name:      image + ":" + imageTag,
```

Comment:
Why you don't use the imageContext in the name ?

Reply:
This is used in go templates to populate ImageStream Trigger, which can't use context in name - 
https://github.com/desmax74/kie-cloud-operator/blob/2ea0b5a8698e18a728b1aad34644ff70bfe478c4/config/7.9.0/common.yaml#L582-L590

Adding imageContext here was the reason @jakubschwan was having issues testing it ... as stated as BUG here - 
https://github.com/kiegroup/kie-cloud-operator/pull/460#issuecomment-670475055

Reply:
imageContext IS used in imageStream itself however, when creating the imageStream.